Base/usr/share/man/man1/test.md

```diff
@@ -0,0 +1,86 @@
+## Name
+
+test - checks files and compare values
+
+## Synopsis
+
+```**sh
+$ test EXPRESSION
+```
+
+## Description
+
+`test` takes a given expression and returns 0 if the expression evaluates to true, or 1 if not.
+
+An omitted expression defaults to false, and an unexpected error causes a return value of 126.
+
+The expression can take any of the following forms:
+
+* `( EXPRESSION )` (grouping)
+
+Boolean operations
+
+* `! EXPRESSION` (negation)
+* `EXPRESSION -a EXPRESSION` (and)
+* `EXPRESSION -o EXPRESSION` (or)
+
+String comparison
+
+* `-n STRING` (non-empty)
+* `-z STRING` (empty)
+* `STRING = STRING` (equality)
+* `STRING != STRING` (unequality)
+* `STRING` (non-empty, if the only expression available)
+
+Integer comparison
+
+* `INTEGER -eq INTEGER` (equality)
+* `INTEGER -ne INTEGER` (unequality)
+* `INTEGER -lt INTEGER` (less-than)
+* `INTEGER -gt INTEGER` (greater-than)
+* `INTEGER -le INTEGER` (less-than-or-equal)
+* `INTEGER -ge INTEGER` (greater-than-or-equal)
+
+File comparison
+
+* `FILE -ef FILE` (same file - same device and inode number)
+* `FILE -nt FILE` (newer-than - checks modification date)
+* `FILE -ot FILE` (older-than - checks modification date)
+
+File type checks
+
+* `-b FILE` (is block device?)
+* `-c FILE` (is character device?)
+* `-f FILE` (is regular file?)
+* `-d FILE` (is directory?)
+* `-p FILE` (is pipe?)
+* `-S FILE` (is socket?)
+* `-h FILE` (is symbolic link?)
+* `-L FILE` (is symbolic link?)
+
+File permission checks
+
+* `-r FILE` (is readable?)
+* `-w FILE` (is writable?)
+* `-x FILE` (is executable?)
+
+File existence check
+
+* `-e FILE` (does file exist?)
+
+
+Except for `-h/-L`, all FILE checks dereference symbolic links.
+
+
+## Options
+
+None.
+
+## Examples
+
+```sh
+# Conditionally do something based on the value of a variable
+$ test "$foo" = bar && echo foo is bar
+# Check some numbers
+$ test \( 10 -gt 20 \) -o \( ! 10 -ne 10 \) && echo "magic numbers!"
```

Comment:
Do the `\`s get srendered properly by LibMarkdown, by the way?

Reply:
Since the `\`s are in a code block (\`\`\`), yes

Reply:
Heh, I know what a code block is; I'm wondering if I messed up while implementing them in LibMarkdown.

Reply:
yes, it's rendered properly.

Reply:
Awesome 🙂